### PR TITLE
code cleanup

### PR DESCRIPTION
**Pull request by RyWhal**
This is a first pass at cleaning things up for readability. In this first pass:
1) Removed everything that is unnecessary to your code (pycache, DS_Store, other drivers, waveshare test images, etc.)
2) I added a .gitignore so that unnecessary files don't get added back
3) I moved non-code stuff into a `extras` directory (parts list, stl, sample config, etc.). README stays in the main directory. 

Note: The code without any errors, but THIS IS NOT TESTED. Since I have the 4.2in rev2.2 display, it doesn't work with my display. But there was no functional changes to the code or drivers. So if anything doesn't work, its likely just an import error. 

I have another branch going thats going to take a lot more work that is a more comprehensive re-org of the code to split functions out a little more logically. But that will take a lot more time. 

in short; I deleted unnecessary things and TEST before merging.

### CODE REVIEW

**Comment by tallzilla** [comment]
This will fix https://github.com/zerowriter/zerowriter1/issues/7, too.

**Comment by RyWhal** [comment]
This wont work with the 2.2 branch update. I'll make a new one once that branch is merged.
